images/macos/scripts/build/install-openssl.sh

```diff
@@ -6,20 +6,26 @@
 
 source ~/utils/utils.sh
 
-echo "Install openssl@3"
-brew_smart_install "openssl@3"
+echo "Install openssl@1.1"
```

Comment:
You have to love downgrades without an information why :( https://github.com/actions/runner-images/commit/84c4b32b94fceb62a57609a34527a3016ec43079

Reply:
Well at least the macos build is not used by us.. for now 😅